docs/guide/troubleshooting.md

```diff
@@ -43,16 +43,24 @@ kubectl port-forward -n "${APPLICATION_NAMESPACE}" \
 
 Then navigate to `localhost:9901/` for the index or `localhost:9901/config_dump` for the envoy config.
 
+## VirtualGateway - Common Issues
+```
+"Error from server (found multiple matching virtualGateways for namespace: namespace, expecting 1 but found N"
```

Comment:
I guess this is no longer valid with the new GatewayRoute selector changes introduced in this diff...